crypto/keyring_message.go

```diff
@@ -228,7 +233,14 @@ func asymmetricDecryptStream(
 		privKeyEntries = append(privKeyEntries, additionalEntries...)
 	}
 
-	config := &packet.Config{Time: getTimeGenerator()}
+	config := &packet.Config{
+		Time: func() time.Time {
+			if verifyTime == 0 {
+				return time.Now()
```

Comment:
I assumed that falling back to device time in this case is ok, since we anyway remove the signature error afterward

Reply:
Technically speaking, this is true for `ErrSignatureExpired`, but not `ErrKeyExpired` (not entirely sure if that's intended, but for now it seems safer to fall back to the server time still, if possible)

Reply:
Ah yes I forgot we check that as well.

Reply:
I guess that depends on the intention of verifyTime=0.
Because when verifyTime=<some time> we act as if everything was happening at that time, so key expiration as well.
But when it's 0, we ignore only the expiration of the sig, which is a bit strange.

Reply:
Yeah, I agree, in openpgp.js, passing `null` would ignore both key expiration and signature expiration. But, that being said, I think ignoring key expiration is a little bit dangerous, so unless someone needs it I wouldn't push very hard for changing this